tests/__init__.py

```diff
@@ -1,3 +1,4 @@
+import pickle
```

Comment:
I preferably don't want to mention `pickle` in this package since it suggests that this would be a sane choice. A better example would be JSON with a custom `JSONEncoder`/`JSONDecoder`. This would then also ensure that the given serialization is actually being used (since the default encoder would throw an error).